sdk/python/tests/conftest.py

```diff
@@ -126,10 +126,16 @@ def universal_data_sources(environment):
 def e2e_data_sources(environment: Environment):
     df = create_dataset()
     data_source = environment.data_source_creator.create_data_source(
-        df, environment.feature_store.project, field_mapping={"ts_1": "ts"},
+        df, destination_name="data_source", field_mapping={"ts_1": "ts"},
```

Comment:
Why are we changing the destination name to `"data_source"`?

Reply:
didn't want this name to conflict with the "entityless_data_source" below, wanted more consistent naming than just reusing the project name

Reply:
Can you leave this one as is, and change the one down below to `{project}.entityless_data_source`? That will ensure that we're trying these data sources to the corresponding project, which is necessary to track down failures and such.

Reply:
In fact, I think that we should leave this data set as is and add the entityless datasources to the universal data soures (line 111).

We should then update `test_historical_retreival` and `test_universal_online` to look up values from there.

The e2e datasets have specific timestamps to assert point in time correctness, and I don't know if that needs to be validated for entityless features, since we're always looking at the latest state of the features (but please correct me if I'm wrong).

Reply:
Yeah, I was debating whether `test_historical` and `test_online` would be enough coverage for this case - I think your explanation makes sense though. I'll remove changes to the universal_e2e